packages/clerk-js/src/core/clerk.ts

```diff
@@ -1160,48 +1153,78 @@ export default class Clerk implements ClerkInterface {
     }
 
     const satelliteUrl = getClerkQueryParam(CLERK_SATELLITE_URL);
-    if (!satelliteUrl) {
-      return false;
-    }
-
-    if (!isHttpOrHttps(satelliteUrl)) {
-      clerkRedirectUrlIsMissingScheme();
-    }
-
-    return true;
+    return !!satelliteUrl;
   };
 
   #syncWithPrimary = async () => {
-    if (this.#shouldSyncWithPrimaryInDevelopment()) {
+    if (this.instanceType === 'development') {
       await this.navigate(this.#buildSyncUrlForDevelopmentInstances());
-    } else if (this.#shouldSyncWithPrimaryInProduction()) {
+    } else if (this.instanceType === 'production') {
       await this.navigate(this.#buildSyncUrlForProductionInstances());
     }
   };
 
+  #validateMultiDomainOptions = () => {
+    if (!this.isSatellite) {
+      return;
+    }
+    if (this.#instanceType === 'development' && !this.#options.signInUrl) {
+      clerkMissingSignInUrlAsSatellite();
+    }
+
+    if (!this.proxyUrl && !this.domain) {
+      clerkMissingProxyUrlAndDomain();
+    }
+  };
+
   #loadInStandardBrowser = async (): Promise<boolean> => {
-    // Dev Browser handling
+    /**
+     * 1. Create the devBrowser.
+     * At this point the devBrowser is not yet setup, but its API is ready for use
+     */
     this.#devBrowserHandler = createDevBrowserHandler({
       frontendApi: this.frontendApi,
       fapiClient: this.#fapiClient,
     });
 
+    /**
+     * 2. Multidomain SSO handling
+     * If needed the app will attempt to sync with another app hosted in a different domain in order to acquire a session
+     * - for development instances it populates dev browser JWT and `devBrowserHandler.setup()` should not have run.
+     */
+    this.#validateMultiDomainOptions();
+    if (this.#shouldSyncWithPrimary()) {
+      await this.#syncWithPrimary();
+      // ClerkJS is not considered loaded during the sync/link process with the primary domain
+      return false;
+    }
+
+    /**
+     * 3. Setup dev browser.
+     * This is not needed for production instances hence the .clear()
+     * At this point we have already attempted to pre-populate devBrowser with a fresh JWT, if Step 3 was successful this will not be overwritten
```

Comment:
Please also elaborate on this by stating that in a satellite app we don't want to get the JWT from FAPI but from URL Based Session Syncing as we are trying to sync the primary and satellite dev browser values.